graphql-kotlin-types/README.md

```diff
@@ -0,0 +1,17 @@
+# graphql-kotlin-types
+[![Maven Central](https://img.shields.io/maven-central/v/com.expediagroup/graphql-kotlin-types.svg?label=Maven%20Central)](https://search.maven.org/search?q=g:%22com.expediagroup%22%20AND%20a:%22graphql-kotlin-types%22)
+[![Javadocs](https://img.shields.io/maven-central/v/com.expediagroup/graphql-kotlin-types.svg?label=javadoc&colorB=brightgreen)](https://www.javadoc.io/doc/com.expediagroup/graphql-kotlin-types)
+
+The core types used by both client and server for communication over GraphQL. These types are annotated with [Jackson](https://github.com/FasterXML/jackson) metadata
+so they can be serialized/deserialized. Instead of using this package, you may want to look at the other fully featured client and server packages in our [main README](./../README.md).
+
+### GraphQLRequest
+Provides the common format for comminicating with [GraphQL over HTTP](https://graphql.org/learn/serving-over-http/#post-request). This format is not part of the specification but it mostly standard at this point in all server implementations as the expected format.
+
+### GraphQLResponse
+Provieds a [GraphQL specification compliant response](http://spec.graphql.org/June2018/#sec-Data) class. This can be used by clients or servers. We also needed to create a custom error class for proper serialization.
```

Comment:
```suggestion
Provides a [GraphQL specification compliant response](http://spec.graphql.org/June2018/#sec-Data) class. This can be used by clients or servers. We also needed to create a custom error class for proper serialization.
```